sdk/cosmosdb/cosmos/src/request/TimeoutError.ts

```diff
@@ -7,7 +7,7 @@ export const TimeoutErrorCode = "TimeoutError";
 
 export class TimeoutError extends Error {
   public readonly code: string = TimeoutErrorCode;
-  constructor(message?: string) {
+  constructor(message: string = "Timeout Error") {
```

Comment:
Is the information about the time. i.e 4000ms etc available while constructing this error. If so, we can add this information here.

Reply:
updated @v1k1